packages/react-components/react-infobutton/src/components/InfoLabel/InfoLabel.types.ts

```diff
@@ -1,6 +1,6 @@
 import { Label } from '@fluentui/react-label';
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 import { InfoButton } from '../InfoButton';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 import type { InfoButtonProps } from '../InfoButton';
 
 export type InfoLabelSlots = {
```

Comment:
mmm I was thinking only the components, but now that you mention, if someone is recomposing they probably are using only types and hooks. I'll deprecate everything exported to avoid such cases.